pkgs/build-support/rust/default.nix

```diff
@@ -11,6 +11,7 @@
 , buildInputs ? []
 , nativeBuildInputs ? []
 , cargoUpdateHook ? ""
+, cargoCopyLockfile ? false
```

Comment:
Is there a better name for this flag though? Copying is an implementation detail but not what this feature is about.

Reply:
Naming is so hard... How about `cargoVerifyHash` ?

Reply:
`verifyCargoLock`?

Reply:
`checkCargoDeps`?

Reply:
Ok I renamed it to `verifyCargoDeps` :)

Reply:
Ok I renamed it to `verifyCargoDeps`. The Cargo.lock is just an implementation detail to verify that the cargoDeps is not out of date.